content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -925,6 +964,8 @@ For watch, the semantics of resource version are:
 
 **Watch:**
 
+{{< table caption="resourceVersion for watch" >}}
```

Comment:
This should have a `{{< /table >}}` at the end of the Markdown table.

Reply:
done.